ddtrace/debugging/_probe/remoteconfig.py

```diff
@@ -119,13 +121,32 @@ def probe(_id, _type, attribs):
     raise ValueError("Unknown probe type: %s" % _type)
 
 
+_SNAPSHOT_PREFIX = "snapshotProbe_"
+_METRIC_PREFIX = "metricProbe_"
+_CONFIG_REGEX = re.compile(r"^[\da-f]{8}-([\da-f]{4}-){3}[\da-f]{12}$", re.IGNORECASE)
```

Comment:
Can we leave an example of a config ID as a comment here?

Reply:
We will be able to do once the old configuration get a prefix as well. but currently we need to make sure that different prefixes are not been treated as service config.